src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/mapper/induction/QualificationsAndTrainingResourceMapper.kt

```diff
@@ -77,8 +80,9 @@ abstract class QualificationsAndTrainingResourceMapper {
   fun toOffsetDateTime(instant: Instant?): OffsetDateTime? = instant?.atOffset(ZoneOffset.UTC)
 
   @Mapping(target = "reference", source = "request.id")
+  @Mapping(target = "qualifications", source = "request.qualifications")
   abstract fun toUpdatePreviousQualificationsDto(
-    request: UpdateEducationAndQualificationsRequest?,
+    request: UpdateEducationAndQualificationsRequest,
```

Comment:
`UpdateEducationAndQualificationsRequest` contains previous training, which can never be null (since at the very least `NONE` has to be provided).